src/style.css

```diff
@@ -51,6 +46,11 @@ body {
   pointer-events: none;
 }
 
```

Comment:
- [ ] Your _functions_ are **not separated** into different files. Kindly ensure that there is a **new JavaScript file** for the new functionality.